### DISCUSSION TITLE
Swap parts of the model

### DISCUSSION THREAD

**Discussion by im-mike** [google/model-viewer#2017] [General]
Hello

Is it possible to change only some parts of the model, for example if we have a model of a human head, can we swap just the glasses for a different glasses model with a dropdown or a button click?

Many thanks
Mike

**Comment by elalish** [comment]
This is what Blender and such are for. Eventually we'd like the scene-graph API to grow to be able to hide/show meshes in a glTF, which would make this more interactable.
